google-apps-script.js

```diff
@@ -21,14 +25,23 @@ function doPost(e) {
   try {
     Logger.log(e); // the Google Script version of console.log see: Class Logger
     record_data(e);
-
-    var mailData = e.parameters; // just create a slightly nicer variable name for the data
+    
+    // shorter name for form data
+    var mailData = e.parameters;
+    
+    // names and order of form elements
+    var dataOrder = JSON.parse(e.parameters.formDataNameOrder);
+    
+    // determine recepient of the email
+    // if you have your email uncommented above, it uses that `TO_ADDRESS`
+    // otherwise, it defaults to the email provided by the form's data attribute
+    var sendEmailTo = (typeof TO_ADDRESS !== "undefined") ? TO_ADDRESS : mailData.formGoogleSendEmail;
```

Comment:
Hmmm, fair point! I was thinking that the default support wouldn't be arbitrary / customizable by the end user directly, but anyone can technically alter the HTML and JS to do what they want! So I was thinking from an attacker perspective. Granted, attackers could spam email to you in many other ways without this script, so perhaps that is moot to avoid feature support for fear alone!